core/libraries/plugin_api/EE_Register_CPT.lib.php

```diff
@@ -254,16 +255,14 @@ public static function default_terms(EE_Register_CPTs $cpt_class)
     /**
      * This deregisters whats been registered on this class (for the given slug).
      *
-     * @since 4.5.0
-     *
      * @param string $cpt_ref The reference for the item registered to be removed.
      *
      * @return void
+     * @since 4.5.0
+     *
      */
-    public static function deregister($cpt_ref = null)
+    public static function deregister(string $cpt_ref = '')
     {
-        if (! empty(self::$_registry[ $cpt_ref ])) {
-            unset(self::$_registry[ $cpt_ref ]);
-        }
+        unset(self::$_registry[ $cpt_ref ]);
```

Comment:
you never need to perform `isset()` or `empty()` checks prior to using `unset()`